birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -24,6 +24,26 @@ if os.getenv("JUPYTERHUB_CRYPT_KEY"):
     c.MagpieAuthenticator.refresh_pre_spawn = True
     c.MagpieAuthenticator.auth_refresh_age = int("${JUPYTERHUB_AUTHENTICATOR_REFRESH_AGE}")
 
+    # Allow users to access their own auth_state in order to get their own magpie cookie
+    # See https://github.com/jupyterhub/jupyterhub/issues/3588 for details
+    c.JupyterHub.load_roles = [
```

Comment:
I assume `JUPYTER_API_TOKEN` now exists because of this new config?

Reply:
No I think it always existed in the environment